cpg-language-python/src/main/kotlin/de/fraunhofer/aisec/cpg/frontends/python/PythonLanguageFrontend.kt

```diff
@@ -157,7 +157,8 @@ class PythonLanguageFrontend(language: Language<PythonLanguageFrontend>, ctx: Tr
                         parseName(id)
                     } else {
                         // If it is not, we want place it in the current namespace
-                        scopeManager.currentNamespace.fqn(id)
+                        // scopeManager.currentNamespace.fqn(id)
+                        id
```

Comment:
Some explanation, please? I cannot say which of the two is correct from reading the code.

Reply:
I have extracted this into a separate PR: https://github.com/Fraunhofer-AISEC/cpg/pull/1684

Reply:
And then closed the other PR. I still don't get the change (and cannot tell which of the two versions is correct), but I'm sure you know what you are doing here...